jenkins-pipelines/manager/centos-manager-backup-azure.jenkinsfile

```diff
@@ -5,11 +5,14 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 managerPipeline(
     backup_bucket_backend: 'azure',
-    backend: 'aws',
+    backend: 'azure',
```

Comment:
jenkinsfile file name shows `centos` while on Azure we release only Ubuntu for now.

Reply:
Centos refers to the OS of the monitor, not the DB